### ISSUE TITLE
Full width control in container

### ISSUE THREAD

**Issue by 79ho3ein** [elementor/elementor#19766]
### Prerequisites

- [X] I have searched for similar features requests in both open and closed tickets and cannot find a duplicate.
- [X] The feature is still missing in the latest stable version of Elementor ( Elementor Pro )

### What problem is your feature request going to solve? Please describe.

Hello
In the container, when we select full width, there is still a custom width control. But is there a need to have custom width control for the full width?

![Uploading container.png…]()

### Describe the solution you'd like

I think it would be much better to hide the custom width control when we set the container width to full width. Just like the video below.

https://user-images.githubusercontent.com/77163248/189524390-959f79e2-d2d7-4fac-8583-76ad9671845e.mp4

**Comment by 79ho3ein** [comment]
I am completing this request and I request you, dear developers, if you are reading this request, please tell me what you think about it. @rami-elementor

Isn't it better to remove the Content Width control in the container? Actually, only one width control (which is already there) is enough for the container. 
Every container we add has a default width. (As you know, the default width is obtained from site settings > layout)
If we want to have a full width container, just set the width to 100%.  Therefore, the Content Width option will not really be needed.

In the video below, you can see that by removing the Content Width option, we will achieve all the desired results.

https://user-images.githubusercontent.com/77163248/189525452-6acc1415-0ea0-4bbb-98d7-6a325d117f1b.mp4


**But here is a very important point.**

- In the Elementor container, the box width determines the width of the content, not the width of the container. In fact, in the video above, if I set the width to 1200px, the container will still be full width. Isn't this a problem?
- 
I checked this in the competing page builder, when I set the width of the container to 1200 pixels, this value was applied to the width of the container, not to the content of the container.

thank you


(Related to #19758)

**Comment by pauserratgutierrez** [comment]
Add auto, fit-content and min and max widths too!!!

**Comment by pauserratgutierrez** [comment]
> I am completing this request and I request you, dear developers, if you are reading this request, please tell me what you think about it. @rami-elementor
> 
> Isn't it better to remove the Content Width control in the container? Actually, only one width control (which is already there) is enough for the container. 
> Every container we add has a default width. (As you know, the default width is obtained from site settings > layout)
> If we want to have a full width container, just set the width to 100%.  Therefore, the Content Width option will not really be needed.
> 
> In the video below, you can see that by removing the Content Width option, we will achieve all the desired results.
> 
> https://user-images.githubusercontent.com/77163248/189525452-6acc1415-0ea0-4bbb-98d7-6a325d117f1b.mp4
> 
> 
> **But here is a very important point.**
> 
> - In the Elementor container, the box width determines the width of the content, not the width of the container. In fact, in the video above, if I set the width to 1200px, the container will still be full width. Isn't this a problem?
> 
> I checked this in the competing page builder, when I set the width of the container to 1200 pixels, this value was applied to the width of the container, not to the content of the container.
> 
> thank you
> 
> 
> (Related to #19758)

Yes, you need to put the container inside a container to set the width...

**Comment by nicholaszein** [comment]
Hey @79ho3ein,

I just stumbled upon this issue, and I think we already discussed it elsewhere. I just can't find where.

This is not something we are planning to remove because this option is used for inner containers, where the width is very important to be manipulated, even if the inner container is set to full width. And we will not make it conditionally hide if you're using a parent container either since this will break the consistency of usage and workflow.

Best regards
